SecurityService.OpenIdConnect.IntegrationTests/Common/Hooks.cs

```diff
@@ -58,15 +60,29 @@ public void AfterScenario()
         [BeforeScenario(Order = 0)]
         public async Task BeforeScenario()
         {
-            ChromeOptions options = new ChromeOptions();
-            options.AddArguments("--disable-gpu");
-            options.AddArguments("--no-sandbox");
-            options.AddArguments("--disable-dev-shm-usage");
-            var experimentalFlags = new List<String>();
-            experimentalFlags.Add("same-site-by-default-cookies@2");
-            experimentalFlags.Add("cookies-without-same-site-must-be-secure@2");
-            options.AddLocalStatePreference("browser.enabled_labs_experiments", experimentalFlags);
-            this.WebDriver = new ChromeDriver(options);
+            String? browser = Environment.GetEnvironmentVariable("Browser");
+            //browser = "Firefox";
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6248858)